File: python/array_problem/h_index.py




def get_h_index(nums):

	nums.sort(reverse= True)

	h_index = 0

	for k,v in enumerate(nums):
		
		if v >= (k + 1):

		   h_index = k + 1

		else:
			
			break

	return h_index

File: js/src/ArrayProblem/HIndex.js




export const HIndex = (list)=>{

	list.sort((a,b)=> b - a);

	let hIndex = 0;

	for (let i=0; i < list.length; i++) {
		
		if(list[i] >= (i + 1)){

			hIndex = (i+1);

		}else{

			break;
		}

	}

	return hIndex;
}